hugr-core/src/builder/handle.rs

```diff
@@ -87,17 +87,7 @@ impl From<BuildHandle<DfgID>> for BuildHandle<CaseID> {
     fn from(value: BuildHandle<DfgID>) -> Self {
         Self {
             node_handle: value.node().into(),
-            num_value_outputs: value.num_value_outputs,
-        }
-    }
-}
-
-impl From<BuildHandle<DfgID>> for BuildHandle<TailLoopID> {
```

Comment:
Could we keep the implementation (dropping the loop flag output)?

Reply:
there may be some outputs embedded in the loop flag output in the "break" variant of the Sum.